e2e/playwright/search/src/tests/search-results-libraries.spec.ts

```diff
@@ -0,0 +1,237 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import {
+  ApiClientFactory,
+  Utils,
+  test,
+  TrashcanApi,
+  NodesApi,
+  SitesApi,
+  FileActionsApi,
+  SITE_VISIBILITY,
+  SITE_ROLES
+} from '@alfresco/playwright-shared';
+
+test.describe('Search Results - General', () => {
+  let trashcanApi: TrashcanApi;
+  let nodesApi: NodesApi;
+  let sitesApi: SitesApi;
```

Comment:
can be defined in beforeAll and can be const there as used only there

Reply:
It cant be defined in beforeAll, because they are used in afterAll too. Only sitesApi can be put in beforeAll, but I would stick with defining everything in one place.

Reply:
Are you sure? I don't see sitesApi in afterAll